src/components/Integrations/Table/IntegrationTest.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from 'react';
+import axios from 'axios';
+import { Button, Modal, TextInput } from '@patternfly/react-core';
+import { useNotification } from '../../../utils/AlertUtils';
+
+const IntegrationTestModal = ({ integrationUUID, isModalOpen, onClose }) => {
+  const [inputValue, setInputValue] = useState('');
+  const { addSuccessNotification, addWarningNotification } = useNotification();
+
+  const placeholderText =
+    'Congratulations! The integration you created on https://console.redhat.com was successfully tested!';
+
+  const failedTestText =
+    'Your integration test has unfortunately failed, please verify your integration information.';
+
+  const handleModalCancel = () => {
```

Comment:
Why wrap a function in a function? It doesn't seem necessary.

Reply:
Ah nice catch!  Pushed logic to the parent, never got rid of the original func lol

I'll clean it up in a sec.